components/Leaderboard/Campaigns/index.jsx

```diff
@@ -48,7 +181,25 @@ export const Campaigns = () => {
       <Table
         columns={columns}
         dataSource={twitterCampaigns}
+        expandable={{
+          expandIcon: ({ onExpand, record }) => {
+            const Icon = ReadOutlined;
+            return (
+              <div
+                style={{ fontSize: '14px', color: '#7E22CE', cursor: 'pointer' }}
+                onClick={(e) => onExpand(record, e)}
+              >
+                <Icon /> Guide
```

Comment:
yeah agree with Mohan, actually can be just
```suggestion
                <ReadOutlined /> Guide
```